code/modules/modular_computers/file_system/programs/arcade.dm

```diff
@@ -100,6 +102,9 @@
 	if(computer)
 		printer = computer.all_components[MC_PRINT]
 
+	SEND_SIGNAL(usr, COMSIG_ADD_MOOD_EVENT, "gaming", /datum/mood_event/gaming)
```

Comment:
You might be able to just put this in `played_game`, right?

Reply:
I wasn't sure if that was the best way to go about it, or if placing it in its own proc and registering it under the `COMSIG_MOB_PLAYED_VIDEOGAME` signal would've been better.

Reply:
Mood itself should do that, but that's a huge refactor that I kinda wanna do later